eval.py

```diff
@@ -1141,17 +1141,19 @@ def play_video():
     cleanup_and_exit()
 
 
-def load_benchmark_batch(dataset, image_idx):
+def load_benchmark_batch(dataset, image_idx, use_cuda = False):
     img = dataset[image_idx]['arr_0']
     TARGET_SIZE = 550, 550
-    img = cv2.resize(img, TARGET_SIZE)
+    img = cv2.resize(img.transpose(1,2,0), TARGET_SIZE)
     img = np.moveaxis(img, -1, 0) # put channels in front
     _, h, w = img.shape
     batch = torch.from_numpy(np.ascontiguousarray(img[np.newaxis], dtype=np.float32))
+    if use_cuda:
+        batch = batch.to(device)
```

Comment:
This is currently sending the batch to cpu. I'm assuming the intended behavior is to send to GPU?

Reply:
This information is being passed from L1483 to `evaluate()` function, down to `load_benchmark_batch()`. I agree however, that setting the flag to True by default, is more adequate, as we are most likely expect to run on cuda.

Reply:
I meant that device is defined as `"cpu"`, so that even if `use_cuda` is `True`, it would be keeping the batch on cpu.